### ISSUE TITLE
Really bad artifacts in the game. None of the troubleshooting solutions work. 

### ISSUE THREAD

**Issue by JennaTulwarts** [runelite/runelite#12763]
**Describe the bug**
Using MSAA above 2x is causing graphical artifacts I have never seen before. They look like the cards from Windows solitaire after a win. I've been playing OSRS on this rig for a little over a year and set up my GPU settings day 1 at 16x and haven't touched anything in that tab since. The rig has been exactly the same hardware wise the entire time with no upgrades or changes to any of my components. I have uninstalled and reinstalled my GPU drivers, reinstalled Runelite, and also rolled back my GPU drivers with zero difference. It's playable with the GPU on and below 2x MSAA but I prefer the clean edges of the 16x. I'm trying to figure out what would cause this and if there was anything I possibly could have done to start this. It concerns me as I'm worried it could possibly affect my other games if this is something about my hardware/software that has for whatever reason stopped functioning how I intend it to. 

**Screenshots**
![RS art](https://user-images.githubusercontent.com/74166150/98498092-bfe61680-21fa-11eb-8754-884389888efe.PNG)

**Environment (please complete the following information):**
 - OS: Windows 10
 - RuneLite version: 1.6.30
 - Launcher version: 2.1.3

**Additional context**
If there is anything else I can provide to help bring clarity to the problem please ask. 

I am using a 1070ti that I did have lightly OC'd. Once this problem started I stopped the OC and still behaves the same way.

**Comment by Nightfirecat** [comment]
This is a known issue on some setups, [listed on our GPU FAQ page under the "I have weird lines all over the screen" heading](https://github.com/runelite/runelite/wiki/GPU-FAQ#i-have-weird-lines-all-over-the-screen)

**Comment by JennaTulwarts** [comment]
This didn't fix it, it just completely disabled my GPU options. Why are
they not working when they have for a full year is what I'm trying to
figure out? I'm not looking for an amputate the arm solution I'm looking
for a save the arm solution.

**Comment by JennaTulwarts** [comment]
I tried to edit the post with the actual solution but you removed it. My
Nvidia Control Panel had a setting on which overridded my AA settings
within whatever game I played. I turned to be application specific and no
more artifacts. I'm very disappointed that your "solution" was to make it
inaccessible as it didn't do anything. Again I equate it to going to the
doctor with some pain in your leg so they amputate it. Look! No more leg
pain! AA problems? No more GPU! See? Fixed!
